src/material/core/typography/_typography.scss

```diff
@@ -96,7 +96,7 @@
   // and h6 at 0.67em.
   .mat-h5, #{$selector} h5 {
     @include mat-typography-font-shorthand(
-      mat-font-size($config, body-1) * 0.83,
+      calc(#{mat-font-size($config, body-1)} * 0.83),
```

Comment:
Can you add a comment that explains the use of `calc` here? I could see someone removing it without considering CSS variables.

Reply:
Sure